kubefirst backupSSL stoped to work as expected

Bucket seems to not be created:
![image](https://user-images.githubusercontent.com/53096417/183131762-328b8c1d-3c4c-4cf1-9480-a8dbc6734c3d.png)

Bucket is missing on AWS console for the certs

Logs:

```bash 
LOG: 2022/08/05 17:38:29.644022 /home/developer/app/internal/aws/aws.go:340: createBucketCalled
LOG: 2022/08/05 17:38:29.654626 /home/developer/app/internal/aws/aws.go:344: creating bucket k1-6zray.kubefirst.tech
LOG: 2022/08/05 17:38:29.655408 /home/developer/app/internal/aws/aws.go:347: region is  us-east-1
LOG: 2022/08/05 17:38:30.515311 /home/developer/app/internal/ssl/ssl.go:92: getting certificates
LOG: 2022/08/05 17:38:30.523941 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:31.103843 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:31.268415 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:31.432316 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:31.597548 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:31.767541 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:31.935316 /home/developer/app/internal/ssl/ssl.go:99: /certs/argo-secret.yaml
LOG: 2022/08/05 17:38:32.668223 /home/developer/app/internal/ssl/ssl.go:99: /certs/argocd-server-tls.yaml
LOG: 2022/08/05 17:38:33.526801 /home/developer/app/internal/ssl/ssl.go:99: /certs/atlantis-tls.yaml
LOG: 2022/08/05 17:38:34.273413 /home/developer/app/internal/ssl/ssl.go:99: /certs/chartmuseum-tls.yaml
LOG: 2022/08/05 17:38:35.132499 /home/developer/app/internal/ssl/ssl.go:99: /certs/gitlab-tls.yaml
LOG: 2022/08/05 17:38:35.845872 /home/developer/app/internal/ssl/ssl.go:99: /certs/gitlab-webservice-tls.yaml
LOG: 2022/08/05 17:38:36.549832 /home/developer/app/internal/ssl/ssl.go:99: /certs/vault-tls.yaml
LOG: 2022/08/05 17:38:37.412884 /home/developer/app/internal/ssl/ssl.go:103: getting secrets
LOG: 2022/08/05 17:38:37.414269 /home/developer/app/internal/ssl/ssl.go:40: error: could not create directory "/home/developer/.k1/ssl" - it must exist to continue. error is: mkdir /home/developer/.k1/ssl: file exists
LOG: 2022/08/05 17:38:37.414812 /home/developer/app/internal/ssl/ssl.go:47: getting resources and filtering using jq
LOG: 2022/08/05 17:38:38.200210 /home/developer/app/internal/ssl/ssl.go:47: getting resources and filtering using jq
LOG: 2022/08/05 17:38:38.679316 /home/developer/app/internal/ssl/ssl.go:47: getting resources and filtering using jq
LOG: 2022/08/05 17:38:38.846992 /home/developer/app/internal/ssl/ssl.go:47: getting resources and filtering using jq
LOG: 2022/08/05 17:38:39.013635 /home/developer/app/internal/ssl/ssl.go:47: getting resources and filtering using jq
LOG: 2022/08/05 17:38:39.185069 /home/developer/app/internal/ssl/ssl.go:47: getting resources and filtering using jq
LOG: 2022/08/05 17:38:39.348537 /home/developer/app/internal/ssl/ssl.go:111: /secrets/argo-secret.yaml
LOG: 2022/08/05 17:38:40.433199 /home/developer/app/internal/ssl/ssl.go:111: /secrets/atlantis-tls.yaml
LOG: 2022/08/05 17:38:41.438624 /home/developer/app/internal/ssl/ssl.go:111: /secrets/chartmuseum-tls.yaml
LOG: 2022/08/05 17:38:42.507441 /home/developer/app/internal/ssl/ssl.go:111: /secrets/gitlab-tls.yaml
LOG: 2022/08/05 17:38:43.536052 /home/developer/app/internal/ssl/ssl.go:111: /secrets/gitlab-webservice-tls.yaml
LOG: 2022/08/05 17:38:44.549032 /home/developer/app/internal/ssl/ssl.go:111: /secrets/vault-tls.yaml
LOG: 2022/08/05 17:38:45.610422 /home/developer/app/internal/ssl/ssl.go:116: getting clusterissuers
LOG: 2022/08/05 17:38:45.611792 /home/developer/app/internal/ssl/ssl.go:40: error: could not create directory "/home/developer/.k1/ssl" - it must exist to continue. error is: mkdir /home/developer/.k1/ssl: file exists
LOG: 2022/08/05 17:38:45.612316 /home/developer/app/internal/ssl/ssl.go:50: getting resources
LOG: 2022/08/05 17:38:46.198009 /home/developer/app/internal/ssl/ssl.go:123: /clusterissuers/letsencrypt-prod.yaml
LOG: 2022/08/05 17:38:47.108252 /home/developer/app/internal/ssl/ssl.go:123: /clusterissuers/letsencrypt-staging.yaml
```

--- a/internal/aws/aws.go
+++ b/internal/aws/aws.go
@@ -339,19 +339,38 @@ func DestroyBucketsInUse(destroyBuckets bool) {
 func CreateBucket(dryRun bool, name string) {
 	log.Println("createBucketCalled")
 
-	s3Client := s3.New(GetAWSSession())
+	sess, err := session.NewSessionWithOptions(session.Options{
+		Config: aws.Config{
+			Region: aws.String(viper.GetString("aws.region")),
+		},
+		Profile: viper.GetString("aws.profile"),
+	})
+
+	if err != nil {
+		log.Println("failed to attempt bucket creation ", err.Error())
+		os.Exit(1)
+	}
+
+	s3Client := s3.New(sess)
 
 	log.Println("creating", "bucket", name)
 
 	regionName := viper.GetString("aws.region")
 	log.Println("region is ", regionName)
+
 	if !dryRun {
-		_, err := s3Client.CreateBucket(&s3.CreateBucketInput{
-			Bucket: &name,
-			CreateBucketConfiguration: &s3.CreateBucketConfiguration{
-				LocationConstraint: aws.String(regionName),
-			},
-		})
+		if regionName == "us-east-1" {
+			_, err = s3Client.CreateBucket(&s3.CreateBucketInput{
+				Bucket: &name,
+			})
+		} else {
+			_, err = s3Client.CreateBucket(&s3.CreateBucketInput{
+				Bucket: &name,
+				CreateBucketConfiguration: &s3.CreateBucketConfiguration{
+					LocationConstraint: aws.String(regionName),
+				},
+			})
+		}
 		if err != nil {
 			if awsErr, ok := err.(awserr.Error); ok {
 				switch awsErr.Code() {

--- a/internal/ssl/ssl.go
+++ b/internal/ssl/ssl.go
@@ -82,12 +82,13 @@ func getItemsToBackup(apiGroup string, apiVersion string, resourceType string, n
 
 // GetBackupCertificates create a backup of Certificates on AWS S3 in yaml files
 func GetBackupCertificates() (string, error) {
+	log.Println("GetBackupCertificates called")
+
+	bucketName := fmt.Sprintf("k1-%s", viper.GetString("aws.hostedzonename"))
+	aws.CreateBucket(false, bucketName)
+
 	config := configs.ReadConfig()
 	namespaces := getNamespacesToBackupSSL()
-	log.Println("GetBackupCertificates called")
-	bucketName := fmt.Sprintf("k1-%s", viper.GetString("aws.hostedzonename"))
-	//path := "cert-manager"
-	aws.CreateBucket(false, bucketName)
 
 	log.Println("getting certificates")
 	certificates, err := getItemsToBackup("cert-manager.io", "v1", "certificates", namespaces, "")

